src/test/java/tests/FilterEvalTests.java

```diff
@@ -824,6 +825,40 @@ public void filterLabelMatching() {
         assertFalse(Qualifier.labelMatches(".", "p.high"));
     }
 
+    @Test
+    public void QualifierProcess_useInvalidUsername_getUserWarning() {
+        TurboUser user = new TurboUser(REPO, "fox", "charlie");
+        IModel model = TestUtils.singletonModel(new Model(REPO,
+                new ArrayList<>(),
+                new ArrayList<>(),
+                new ArrayList<>(),
+                new ArrayList<>(Arrays.asList(user))));
```

Comment:
8 space indentation is a last resort. In this case, the following may be better?
```java
IModel model = TestUtils.singletonModel(new Model(REPO,
                                                  new ArrayList<>(),
                                                  new ArrayList<>(),
                                                  new ArrayList<>(),
                                                  new ArrayList<>(Arrays.asList(user))));
```
Alternatively,
```java
IModel model = TestUtils.singletonModel(new Model(REPO, new ArrayList<>(), new ArrayList<>(),
                                                  new ArrayList<>(), new ArrayList<>(Arrays.asList(user))));
```